### ISSUE TITLE
Errors are not logged sometimes in rc16

### ISSUE THREAD

**Issue by vrnvorona** [jonsamwell/flutter_gherkin#256]
Because of try-catch construction in lib/src/flutter/runners/gherkin_integration_test_runner.dart:222 errors are caught but not reported in console (or else reporting methods) which is very annoying.

@jonsamwell can you please test it with adding step without definition?

**Comment by vrnvorona** [comment]
```
for (int i = 0; i < steps.length; i++) {
  try {
    final result = await steps[i](dependencies, hasToSkip);
    if (_isNegativeResult(result.result)) {
      failed = true;
      hasToSkip = true;
    }
  } catch (e) {
    // todo vrnvorona
    print(e);
    failed = true;
    hasToSkip = true;
  }
}
```
If I add print here it works and prints `GherkinStepNotDefinedException: Step definition not found for text:` but without it's silent

**Comment by jonsamwell** [comment]
Thanks, it should be caught and recorded when executing the step.  I'll have time this weekend to go over this and the other issues.

**Comment by jonsamwell** [comment]
Fixed in 3.0.0-rc.17
